setup.py

```diff
@@ -324,10 +386,17 @@ def run(self):
 extra = ["LICENSE", "ThirdPartyNotices.txt", "Privacy.md"]
 
 # Description
-README = path.join(getcwd(), "docs/python/README.rst")
-if not path.exists(README):
-    this = path.dirname(__file__)
-    README = path.join(this, "docs/python/README.rst")
+if(is_openvino):
```

Comment:
can you reduce the duplicated code here?
do something like
readme_file = "docs/python/ReadMeOV.rst" if is_openvino else "docs/python/README.rst"
README = path.join(getcwd(), readme_file)
...